container-runtime/gvisor/runsc-kvm.toml

```diff
@@ -0,0 +1,2 @@
+[runsc_config]
+  platform = kvm
```

Comment:
![image](https://github.com/user-attachments/assets/e86f9971-7ace-45b9-8eb0-5c270c154914)


this looks weird

Reply:
I wonder if it's missing quotes?

Reply:
Yes, I noticed that as well, but I don't understand why. Let me look in a hex editor

Reply:
shouldn't it be quoted?

Reply:
it needs to be quoted

Reply:
Should I add or will you? I think the commit was resigned by one of you

Reply:
I can fix that, i'll add a test on talos side and test before merging